src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/ExplosiveTool.java

```diff
@@ -59,7 +59,7 @@ public boolean onBlockBreak(BlockBreakEvent e, ItemStack item, int fortune, List
 
                         List<Block> blocks = findBlocks(e.getBlock());
 
-                        if (callExplosionEvent.getValue().booleanValue()) {
+                        if (callExplosionEvent.getValue()) {
```

Comment:
Removing this is quite a bad idea, you should always use the primitive boolean here instead of boxing it.